WebUI/Pages/AdminDashboard/AdminGames.razor

```diff
@@ -1,5 +1,163 @@
+        <button class="btn btn-secondary" @onclick="@ToggleGameForm">Cancel</button>
+    </EditForm>
+}
+@* --- Display Game List --- *@
+else if (games != null && games.Count != 0)
+{
+    <div class="row" style="padding-top: 10px">
+        @foreach (var game in games)
+        {
+            <div class="col-sm-6">
+                <div class="card">
+                    <div class="card-body">
+                        <h5 class="card-title">@game.Name</h5>
+                        <p class="card-text">@game.Description</p>
+                        <button @onclick="() => InitEdit(game)" class="btn btn-primary">Edit Game</button>
+                        <button @onclick="() => HandleDelete(game)" class="btn btn-danger">Delete Game</button>
+                    </div>
+                </div>
+            </div>
+        }
+    </div>
+}
+else
+{
+    <br />
+    <h3>No games available... create a new game!</h3>
+}
+
+
+@code {
+
+    #region Declarations
+    private Game selectedGame = new Game();
+    private GameVM adminGameViewModel = new GameVM();
+    private List<Game> games;
+    private List<GameGenre> gameGenres;
+    private bool showForm = false;
+    private bool isAdd = true;
+    #endregion
+
+    //Initializes game list and game genre list
+    protected override async Task OnInitializedAsync()
+    {
+        await FetchGames();
+        gameGenres = await gameGenreService.GetGameGenres();
+    }
+
+    private async Task FetchGames()
+    {
+        games = await gameService.GetGames();
+    }
+
+    #region CRUD Opperations
+    private async Task HandleFormSubmit()
+    {
+        await (isAdd ? AddGameValidSubmit() : EditGameValidSubmit());
+    }
+
+    //Adds new game to DB
+    private async Task AddGameValidSubmit()
+    {
+        Game createGame = GameVM.ToModel(selectedGame, adminGameViewModel);
+        await gameService.AddGame(createGame);
+        await FetchGames();
+        ToggleGameForm();
+        StateHasChanged();
+    }
+
+    //Handles edit operations to selected game record
+    private async Task EditGameValidSubmit()
+    {
+        Game updateGame = GameVM.ToModel(selectedGame, adminGameViewModel);
+        await gameService.UpdateGame(updateGame);
+        await FetchGames();
+        ToggleGameForm();
+        StateHasChanged();
+    }
+
+    private async Task HandleDelete(Game game)
+    {
+        await gameService.DeleteGame(game);
+        await FetchGames();
+        StateHasChanged();
+    }
+    #endregion
+
+    private void InitEdit(Game game)
+    {
+        isAdd = false;
+        selectedGame = game;
+        adminGameViewModel = GameVM.ToViewModel(game);
+        ToggleGameForm();
+    }
+
+    private void InitAdd()
+    {
+        isAdd = true;
+        selectedGame = new Game();
+        adminGameViewModel = GameVM.ToViewModel(selectedGame);
+        ToggleGameForm();
+    }
+
+    #region Visibility Toggles
+    //Toggles visibility of "Add Game" form
```

Comment:
You don't need regions around single methods since they’re collapsable out of the box. 
Also, don't overuse the comments: if the method name is descriptive enough (which it should be), then comments don't bring any value. In this case, I would suggest deleting it.